src/pages/Popup/Popup.jsx

```diff
@@ -1,62 +1,67 @@
-import React, { useEffect } from 'react';
+import React, { useState, useEffect } from 'react';
 import logo from '../../assets/img/logo_light.svg';
 import Greetings from '../../containers/Greetings/Greetings';
 import './Popup.css';
 import '../../assets/fontawesome/css/fontawesome.min.css';
 import '../../assets/fontawesome/css/solid.min.css';
 
 const Popup = () => {
+  // Button logic
+  const [buttonRecordActive, setButtonRecordActive] = useState(false);
+  const [buttonPlayActive, setButtonPlayActive] = useState(false);
+  const [recordText, setRecordText] = React.useState("Record");
```

Comment:
First, why both `React.useState` and `useState`?

Reply:
oh you are right. Let me correct it.